web/themes/custom/droopler_theme/src/components/d-p-single-text-block/_d-p-single-text-block.scss

```diff
@@ -128,6 +125,14 @@
     display: grid;
     height: 100%;
     grid-template-rows: auto min-content min-content 1fr min-content;
+
+    @each $justify in start, center, end {
```

Comment:
Do we need these? Bootstrap utility classes aren't enough for that?

Reply:
It would be possible to add a class to the button container, but it would be much more complicated. This solution is simple.